scripts/equidistant_audio_chunk_sampler.py

```diff
@@ -0,0 +1,132 @@
+"""Audio Frame Sampler.
+
+This script samples audio chunks from audio files and saves them as WAV files.
+
+Usage:
+    python equidistant_audio_sampler.py --audio_paths PATH [PATH ...] --output_dir PATH
+                                       [--chunk_size SAMPLES] [--stride SAMPLES]
+                                       [--target_rate RATE] [--max_frames_per_file LIMIT]
+                                       [--num_chunks SAMPLES]
+
+Arguments:
+    --audio_paths PATH [PATH ...]
+        Paths to one or more audio files. Required.
+
+    --output_dir PATH
+        Directory where sampled chunks will be saved as WAV files. Required.
+
+    --chunk_size SAMPLES
+        Number of samples in each chunk. Default is 16000 (1 second at 16kHz).
+
+    --stride SAMPLES
+        Number of samples to advance between chunks. Default is equal to chunk_size.
+
+    --target_rate RATE
+        Target sample rate for resampling. Default is 16kHz.
+
+    --max_frames_per_file LIMIT
+        Maximum number of frames to process from each file. Default is None.
+
+    --num_chunks SAMPLES
+        Number of chunks to sample across all audio files. Default is 65536 (2^16).
+"""
+
+import argparse
+from pathlib import Path
+
+import torchaudio
+
+from ami.interactions.environments.audio_observation_generator import (
+    AudioFilesObservationGenerator,
+)
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description="Sample chunks from audio files")
+    parser.add_argument(
+        "--audio-paths",
+        nargs="+",
+        required=True,
+        help="Paths to audio files",
+    )
+    parser.add_argument(
+        "--chunk-size",
+        type=int,
+        default=16000,
+        help="Number of samples in each chunk",
+    )
+    parser.add_argument(
+        "--stride",
+        type=int,
+        default=None,
+        help="Number of samples to advance between chunks",
+    )
+    parser.add_argument(
+        "--target-rate",
```

Comment:
```suggestion
        "--sample-rate",
```
`target-rate`だと変数名だけ見た時なんの割合なのかいまいちピンとこないかも　

Reply:
修正
https://github.com/MLShukai/ami/pull/271/commits/b1af5b49b343b11684de593bf53a313d5e2e0243